src/components/ProductList/index.tsx

```diff
@@ -2,14 +2,16 @@ import React from 'react'
 
 const ProductList = () => {
   return (
-      <section className="product-list">
-        <header>
+    <section className="product-list">
+      <a href="/product-detail">
+        <article>
```

Comment:
I have already commented that please make this a `4 x 1` `div` boxes. It's not a  good practice to use `<a>` tag over an `article` since (here)[https://stackoverflow.com/questions/13548225/what-is-htmls-a-a-tag-default-display-type]. You can see that `<a>` tag display is an `inline`. IMO, how about:
```
 section
    article
      header
        title
        paragraph
      a
    article
    article
    article


```

Reply:
And this should become a component called `ProductView`.

Reply:
1. Could you please detail about `4 x 1 div`? 
2. about the `<a>` tag, I've check this article before [https://stackoverflow.com/questions/38367002/make-entire-article-tag-a-link] so that I've done like this. What if we fix this problem by adding `display: inline-block` instead?
3. The code you gave is very interesting, but it's mean we cannot click the title right? or we should back to have `button` to click instead
4. Which component? we already use `ProductView` to show only on the (old) `ProductDetail` page.